src/ctia/http/middleware/unknown.clj

```diff
@@ -0,0 +1,22 @@
+(ns ctia.http.middleware.unknown
+  (:require [ring.util.response :refer [not-found]]))
+
```

Comment:
Could you add comment to explain why this middleware is added?

Reply:
As stated in the comment above:

"I have kept the unknown.clj file (along with the now-unused wrap-unknowns wrapper), as this contains the HTML to be returned. A reason to keep the wrap-unknowns function is if we ever want to have HTML that looks like the previous version returned by ring/compojure. The old 404 response included the requested URI, but the compojure.route/not-found path does not get access to this."